catalog/gitops/configsync/setters.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setters-config
```

Comment:
You want this name to be setters or setters-config?

Reply:
`setters`.